5._Governance.md

```diff
@@ -6,57 +6,71 @@ As a community effort organized within the [Supply Chain Integrity WG](https://g
-* collectively reviewing and revising the roadmap on a biannual basis
+* active participation within the community
+* collectively reviewing and revising the Project mission, vision, strategy, and roadmap every 6 months or more frequently if needed, and publishing Steering Committee meetings notes
+* publishing quarterly updates suitable for OpenSSF TAC and other working groups' consumption
 * participating in strategic planning, such as coordinating face-to-face meetings or suggesting and approving changes to the governance model
-* creating or restructuring workstreams
+* managing the lifecycle of existing and proposed Workstreams within the Project with the consensus input from the community of Participants
+* creating or restructuring Workstreams
 * responding to specific issues or concerns above and beyond the domain of the various workstreams
 * making decisions when community consensus cannot be reached, pursuant to the appeal process documented below
+* holding Maintainers accountable to the standards of what makes SLSA an effective Project and representative of a diverse community of Participants and Contributors. In the extreme event that a majority of existing Maintainers are unavailable to approve additions or removals for an extended period of time, the Steering Committee may add or remove Maintainers to bring the Project back to a functioning state
+* administration of slsa-framework GitHub organization, including maintaining team memberships for Maintainers, setting security defaults such as branch protection for the organization, managing secrets, etc.
+
+The Steering Committee will be comprised of Members from a diverse set of affiliations. An affiliation must not be represented more than once on the Steering Committee. If affiliation changes during a Steering Committee Member's term which would cause overrepresentation, a Steering Committee Member from that affiliation must resign.
+
+
+**1.5. Steering Committee Member Terms.** Steering Committee Members have a one year term.
+One month before the expiration of a Steering Committee Member term, any Participant may submit a nomination to fill the seat. A Participant must include the following documentation in their nomination:
+
+* Statement of intent to perform the duties of a Steering Committee Member during the next term
+* Evidence of active participation in the community and being well-versed in SLSA
+* Affiliation / employer
 
-**1.5. Steering Committee Member Terms.** During the initial year, the Steering Committee Members will agree on grouping themselves into two groups, one to serve an initial two-year term, and the other for an initial one-year term. Thereafter, the Steering Committee Members will serve two year terms.
+The nomination period should last two weeks. After the nomination period, the voting period should begin and last one week. The Steering Committee Members will then tally the votes and announce the new Steering Committee Members.
```

Comment:
Who gets to vote? Just the SC? SC + Maintainers? All project Participants?

Reply:
Thanks Joshua, I added a line that Participants are eligible to vote. Since any Participant can self nominate, having that be the same eligibility for voting seems reasonable.